src/formatter/builder.ts

```diff
@@ -99,28 +105,40 @@ const FormatterBuilder = {
     let Syntax = JavascriptSnippetSyntax
     if (doesHaveValue(snippetSyntax)) {
       const fullSyntaxPath = path.resolve(cwd, snippetSyntax)
-      Syntax = require(fullSyntaxPath) // eslint-disable-line @typescript-eslint/no-var-requires
+      Syntax = FormatterBuilder.resolveConstructor(
+        await importer(pathToFileURL(fullSyntaxPath))
```

Comment:
Why is importer used here always instead of a switch between importer / require? Same for other instances

Reply:
For custom snippet syntaxes and formatters, there should be no need to fall back to `require` - we never supported transpilation for these things, so whether they are ESM or CommonJS modules, the `import` will just work. Support code is only different because of the transpilation.